.azure-pipelines/cleanup.yml

```diff
@@ -0,0 +1,6 @@
+steps:
+- script: |
+    sudo kill -9 `sudo cat /var/run/march/docker.pid` || true
+    sudo rm -f /var/run/march/docker.pid || true
+    sudo rm -rf sonic-buildimage || sudo rm -rf ../sonic-buildimage || true
```

Comment:
the code checked out as /agent/_work/1/s

there is no sonic-buildimage directory here.

should do sudo rm -rf target

Reply:
can we publish $(system.DefaultWorkingDirectory)/target, instead of $(system.DefaultWorkingDirectory)?

Reply:
if $(system.DefaultWorkingDirectory)/fsroot exist, publish maybe failed.